x-pack/plugins/siem/public/components/page/hosts/first_last_seen_host/index.tsx

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiLoadingSpinner } from '@elastic/eui';
+import moment from 'moment';
+import React from 'react';
+import { ApolloConsumer } from 'react-apollo';
+import { pure } from 'recompose';
+
+import { useFirstLastSeenHostQuery } from '../../../../containers/hosts/first_last_seen';
+import { getEmptyTagValue } from '../../../empty_value';
+import { PreferenceFormattedDate } from '../../../formatted_date';
+import { LocalizedDateTooltip } from '../../../localized_date_tooltip';
+
+export type FirstLastSeenHostType = 'first-seen' | 'last-seen';
+
+export const FirstLastSeenHost = pure<{ hostname: string; type: FirstLastSeenHostType }>(
+  ({ hostname, type }) => {
+    return (
+      <ApolloConsumer>
+        {client => {
+          const { loading, firstSeen, lastSeen } = useFirstLastSeenHostQuery(
+            hostname,
+            'default',
+            client
+          );
+          return (
+            <>
+              {loading && <EuiLoadingSpinner size="m" />}
+              {!loading &&
+              type === 'first-seen' &&
+              firstSeen != null &&
+              new Date(firstSeen).toString() === 'Invalid Date'
```

Comment:
In other parts of the code we have been using moment's isValid
http://momentjs.com/docs/#/parsing/is-valid/

It looks more strict:
```ts
new Date(2013, 25, 14).toString();
"Sat Feb 14 2015 00:00:00 GMT-0700 (Mountain Standard Time)"
```

Compared to:

```ts
moment([2013, 25, 14]).isValid()
```